services/net/capture/CaptureAction.cs

```diff
@@ -97,9 +97,15 @@ public CaptureAction(IKafkaMessenger producer, IApiService api, IOptions<Capture
                         reference.Offset = messageResult.Offset;
                     }
 
-                    // Assuming some success at this point, even though a stop command can be called for different reasons.
-                    reference.Status = (int)WorkflowStatus.Received;
-                    await this.Api.UpdateContentReferenceAsync(reference);
+                    // The content service will often already have imported this content before we can update the content reference.
+                    reference = await this.Api.FindContentReferenceAsync(reference.Source, reference.Uid);
+                    if (reference != null)
+                    {
+                        // Assuming some success at this point, even though a stop command can be called for different reasons.
+                        if (reference.Status != (int)WorkflowStatus.Imported)
```

Comment:
If it's already imported, we just want to update the Kafka reference information.